Modules/_zoneinfo.c

```diff
@@ -90,15 +86,19 @@ struct StrongCacheNode {
     PyObject *zone;
 };
 
-static PyTypeObject PyZoneInfo_ZoneInfoType;
+typedef struct {
+    PyTypeObject *ZoneInfoType;
 
-// Globals
-static PyObject *TIMEDELTA_CACHE = NULL;
-static PyObject *ZONEINFO_WEAK_CACHE = NULL;
-static StrongCacheNode *ZONEINFO_STRONG_CACHE = NULL;
-static size_t ZONEINFO_STRONG_CACHE_MAX_SIZE = 8;
+    // Imports
+    PyObject *io_open;
+    PyObject *_tzpath_find_tzfile;
+    PyObject *_common_mod;
 
-static _ttinfo NO_TTINFO = {NULL, NULL, NULL, 0};
+    PyObject *TIMEDELTA_CACHE;
```

Comment:
```suggestion
    // Globals
    PyObject *TIMEDELTA_CACHE;
```

Reply:
They're not globals anymore; the are module level variables :)

Reply:
Yes, this is correct 🤔 
But, right now they are under `// Imports` section. I think it is a good idea to separate them.

Reply:
I added a newline between them for visual separation, but maybe that's not sufficient. Three of the remaining variables are cache pointers. The latter is ... I'm not sure how to classify `NO_TTINFO`. Perhaps best to leave it for Paul.